packages/calypso-e2e/src/lib/pages/marketing-page.ts

```diff
@@ -8,14 +8,12 @@ type MarketingPageTab =
 	| 'Connections'
 	| 'Sharing Buttons'
 	| 'Business Tools';
+type SEOPageTitleStructureCategories = 'Front Page' | 'Posts' | 'Pages' | 'Tags' | 'Archives';
+type SEOExternalServices = 'Google search' | 'Facebook' | 'Twitter';
 type SocialConnection = 'Facebook' | 'LinkedIn' | 'Tumblr' | 'Mastodon' | 'Instagram Business';
 
 const selectors = {
-	// Traffic tab
-	websiteMetaTextArea: '#advanced_seo_front_page_description',
-	seoPreviewButton: '.seo-settings__preview-button',
-	seoPreviewPane: '.web-preview.is-seo',
-	seoPreviewPaneCloseButton: '.web-preview__close',
```

Comment:
**Praise:** You've done such a nice job in these PRs updating the locator syntax as you go! I really appreciate that! 😄

Reply:
![image](https://github.com/Automattic/wp-calypso/assets/6549265/a27dea9d-3fc2-4cf9-be59-abf44634044a)